### ISSUE TITLE
winogrande cannot be dowloaded 

### ISSUE THREAD

**Issue by ghost** [huggingface/datasets#1636]
Hi,
I am getting this error when trying to run the codes on the cloud.  Thank you for any suggestion and help on this @lhoestq 

```
 File "./finetune_trainer.py", line 318, in <module>
    main()
  File "./finetune_trainer.py", line 148, in main
    for task in data_args.tasks]
  File "./finetune_trainer.py", line 148, in <listcomp>
    for task in data_args.tasks]
  File "/workdir/seq2seq/data/tasks.py", line 65, in get_dataset
    dataset = self.load_dataset(split=split)
  File "/workdir/seq2seq/data/tasks.py", line 466, in load_dataset
    return datasets.load_dataset('winogrande', 'winogrande_l', split=split)
  File "/usr/local/lib/python3.6/dist-packages/datasets/load.py", line 589, in load_dataset
    path, script_version=script_version, download_config=download_config, download_mode=download_mode, dataset=True
  File "/usr/local/lib/python3.6/dist-packages/datasets/load.py", line 267, in prepare_module
    local_path = cached_path(file_path, download_config=download_config)
  File "/usr/local/lib/python3.6/dist-packages/datasets/utils/file_utils.py", line 308, in cached_path
    use_etag=download_config.use_etag,
  File "/usr/local/lib/python3.6/dist-packages/datasets/utils/file_utils.py", line 487, in get_from_cache
    raise ConnectionError("Couldn't reach {}".format(url))
ConnectionError: Couldn't reach https://raw.githubusercontent.com/huggingface/datasets/1.1.3/datasets/winogrande/winogrande.py
yo/0 I1224 14:17:46.419031 31226 main shadow.py:122 > Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.6/dist-packages/torch/distributed/launch.py", line 260, in <module>
    main()
  File "/usr/local/lib/python3.6/dist-packages/torch/distributed/launch.py", line 256, in main
    cmd=cmd)
```

**Comment by hungluumfc** [comment]
I have same issue for other datasets (`myanmar_news` in my case).

A version of `datasets` runs correctly on my local machine (**without GPU**) which looking for the dataset at 
```
https://raw.githubusercontent.com/huggingface/datasets/**master**/datasets/myanmar_news/myanmar_news.py
```

Meanwhile, other version runs on Colab (**with GPU**) failed to download the dataset. It try to find the dataset at `1.1.3` instead of` `master`. If I disable GPU on my Colab, the code can load the dataset without any problem.

Maybe there is some problem with the GPU and CPU version of code for these datasets?

**Comment by lhoestq** [comment]
It looks like they're two different issues

----------

First for `myanmar_news`: 

It must come from the way you installed `datasets`.
If you install `datasets` from source, then the `myanmar_news` script will be loaded from `master`.
However if you install from `pip` it will get it using the version of the lib (here `1.1.3`) and `myanmar_news` is not available in `1.1.3`.

The difference between your GPU and CPU executions must be the environment, one seems to have installed `datasets` from source and not the other.

----------

Then for `winogrande`:

The errors says that the url https://raw.githubusercontent.com/huggingface/datasets/1.1.3/datasets/winogrande/winogrande.py is not reachable.
However it works fine on my side.

Does your machine have an internet connection ? Are connections to github blocked by some sort of proxy ?
Can you also try again in case github had issues when you tried the first time ?
